patient-service/src/main/java/com/raf/si/patientservice/service/SchedMedExaminationService.java

```diff
@@ -3,10 +3,16 @@
 import com.raf.si.patientservice.dto.request.SchedMedExamRequest;
 import com.raf.si.patientservice.dto.request.UpdateSchedMedExamRequest;
 import com.raf.si.patientservice.dto.response.SchedMedExamResponse;
+import org.springframework.data.domain.Page;
+
+import java.util.Date;
+import java.util.Optional;
+import java.util.UUID;
 
 public interface SchedMedExaminationService {
 
     SchedMedExamResponse createSchedMedExamination(SchedMedExamRequest schedMedExamRequest);
     SchedMedExamResponse updateSchedMedExaminationExamStatus(UpdateSchedMedExamRequest updateSchedMedExamRequest);
+    Page<SchedMedExamResponse> getSchedMedExaminationByLbz(UUID lbz, Optional<Date> appointmentDate, String token, int page, int size);
```

Comment:
Ista prica kao u kontroleru, mozes da napravis klasu npr. SchedMedExamListResponse koji ce u sebi da ima listu SchedMedExamRepsonse-ova, i jos jedno polje count koje ce predstavljati broj elemenata koji bi se izvukli iz baze. To mozes da izvuces iz Page-a (pogledaj HealthRecordMapper)